README.md

```diff
@@ -20,20 +20,47 @@ Only the latest nightly build is kept.
 Prerequisites
 -------------
 
-#### *nix building
+### *nix building
 
+###### Minimum required packages:
 ```
-g++ make pkg-config libgtk2.0-dev libcanberra-gtk-module libgl-dev libasound2-dev
+g++ make pkg-config libgtk2.0-dev libcanberra-gtk-module libgl-dev libasound2-dev libao-dev libopenal-dev
 ```
 
-By default, GTK2 is used, but support for GTK3 is available. You will need to install the additional package `libgtk-3-dev` as well
-as specifying the command line option `hiro=gtk3` at compile time.
+###### GTK2 & GTK3
+By default, GTK2 is used, but support for GTK3 is available. You will need to install the additional package `libgtk-3-dev` as well as specifying the command line option `hiro=gtk3` at compile time.
 
-#### Windows building
+###### SDL2 for input
+If you would like to use SDL for input, you will need to install the following the `libsdl2-dev` and `libsdl2-0.0` packages and perform a clean build of ares. You should then be able to select SDL for input in the Settings > Drivers menu.   
+
+##### Building with clang
+
+clang++ is now the preferred compiler for ares as it is known to produce a higher performing executable. If it is detected, the build will default to building with clang. It is recommended to install the `clang` package. If you would like to manually specify a compiler, you can use the following option: `compiler=[g++|clang++]`
```

Comment:
We've learned (#982) that for certain Linux Mint and Ubuntu installs, just installing the `clang` package leads to an incomplete build environment. The remediation for that case was highly specific, so I'm not suggesting we repeat it here. I'm really not sure what would be better general advice.

Reply:
The situation with msys2 is even worse; users absolutely should not install the package simply named `clang` (#932). Since this is in the *nix section and you call out specific advice for msys2 below, I think that's fine.

Reply:
Discussed in discord, the additional package for Mint 21 is more of an exception then the rule. We'll leave it as is to avoid having to write detailed instructions for every distro. 

clang on Windows/MSYS is addressed in the Windows section.